auth/src/error.rs

```diff
@@ -27,8 +27,14 @@ pub enum Error {
     IncompleteCheckoutSession,
     #[error("Interacting with stripe resulted in error: {0}.")]
     Stripe(#[from] StripeError),
-    #[error("Missing subscription ID from the checkout session.")]
+    #[error("Missing subscription ID.")]
     MissingSubscriptionId,
+    #[error("found more than one subscription items with the same metadata id: {0}")]
+    DuplicateSubscriptionItems(String),
+    #[error("found no subscription item with the given metadata id: {0}")]
+    MissingSubscriptionItem(String),
+    #[error("stripe subscription is canceled")]
```

Comment:
```suggestion
    #[error("stripe subscription is already canceled")]
```

Reply:
`cancelled`

Reply:
It depends on if you go for American or British English, and Stripe uses the American spelling (one L). But it does feel like a typo to me too, so I'll change it to cancelled.

Reply:
classic dilemma xD, TIL